doc/running/server.md

```diff
@@ -44,7 +44,7 @@ Output availability depends on the type of artifacts generated by the calculatio
 
 ### Run a risk calculation on top of hazard
 
-To run a risk-only calculation on top of a previous hazard run you have to click the `Run Risk` button on the corresponding hazard calculation and select the zip file containing the risk `job.ini` and the related inputs
+To run a risk-only calculation on top of a previous hazard run you have to click the `Continue` button on the corresponding hazard calculation and select the zip file containing the risk `job.ini` and the related inputs
```

Comment:
The whole sentence (and the title) should be changed, explaining as in the subject of this PR.